.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: ruby
 rvm:
-  - 2.2.5
-  - 2.3.1
```

Comment:
Why removed those versions ??

Reply:
This is not remove.

2.2 is 2.2.7
2.3 is 2.3.4

ref: https://travis-ci.org/line/line-bot-sdk-ruby/jobs/396817542

Reply:
I see.
Thank you for letting me know the link.
It is LGTM